### ISSUE TITLE
Incomplete description for joystick service

### ISSUE THREAD

**Issue by xiota** [igo95862/bubblejail#78]
### Output of `bubblejail --version`

AUR-git 0.8.2.r0.g12307c4

### Your distro name and version

Arch

### Description

... required _____.

https://github.com/igo95862/bubblejail/blob/12307c4eedc736f491a652f20aac6667671ce1b1/src/bubblejail/services.py#L620-L624

**Comment by igo95862** [comment]
Should be fixed in 547c4692b85653a0a5774620b31a0ad3158b6ef6.

Thank you for finding this.

**Comment by xiota** [comment]
I use packages built by an automated server.  Apparently it has been down for the last few days, and a package with fix wasn't built.  I've confirmed the issue is fixed in 547c469.

This is a case where the total count vs revisions in the package version string makes a difference.  I know that 0.8.2 was released and the fix for this commit should be in some later commit, but there's no self-contained reference for what count to watch for.  Also semantically, there are not 500+ revisions of 0.8.2.  So a different letter should be used, like `c` (commit, count).
